spec/requests/likes_spec.rb

```diff
@@ -1,7 +1,7 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Likes', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
-end
+# RSpec.describe 'Likes', type: :request do
+#   describe 'GET /index' do
+#     pending "add some examples (or delete) #{__FILE__}"
+#   end
+# end
```

Comment:
- [ ] Well at that point, you can remove the unused testing file. Instead of commenting them.